assets/dev/js/editor/container/container.js

```diff
@@ -419,6 +470,30 @@ export default class Container extends ArgsObject {
 		return false;
 	}
 
+	/**
+	 * Function forSomeChildrenRecursive().
+	 *
+	 * Will run over children recursively, breaks if the callback return true.
+	 *
+	 * @param {function(container:Container)} callback
+	 *
+	 */
```

Comment:
Try to find a better name

Reply:
I have no idea...
@Nevoss, @StyleShit any suggestions?

Reply:
Didn't really understand what it does, but maybe `iterateChildrenRecursive()`?

Reply:
Like in JS `array.some` will loop through array till the callback return positive result, but here it's recursive, runs over all containers in the 'children' hierarchy.

Reply:
someChildrenRecursive @matipojo ?

Reply:
Maybe add to the function description that it's like `array.some()`.

Reply:
Just `some`

Reply:
Then we need to do it over all the `container.js` for the methods with the same 'logical suffix':

```
`findChildrenRecursive()` => `find()`
`forEachChildrenRecursive()` => `forEach()`
`forSomeChildrenRecursive()` => `some()`
```

@matipojo can you approve doing does in this PR?

Reply:
`findChildrenRecursive()` => `recursiveFind()`...

Reply:
And now?